x-pack/plugins/security_solution/common/endpoint/data_generators/exceptions_list_item_generator.ts

```diff
@@ -250,4 +251,33 @@ export class ExceptionsListItemGenerator extends BaseDataGenerator<ExceptionList
       ...overrides,
     };
   }
+
+  generateBlocklist(overrides: Partial<ExceptionListItemSchema> = {}): ExceptionListItemSchema {
+    const blocklist = new BlocklistGenerator().generate();
+    return this.generate({
+      name: blocklist.name,
+      list_id: blocklist.list_id,
+      entries: blocklist.entries,
+      os_types: blocklist.os_types,
+      ...overrides,
+    });
+  }
```

Comment:
I would suggest we do this similar to how we do it for the rest of the artifacts.

Reply:
I'm not sure what you mean here. this is a duplicate of the above `generateHostIsolationException`, `generateEventFilter`, and `generateTrustedApp`.